### PR TITLE
EBP: Improve Manual data fixes page styles

### PR DESCRIPTION
**Pull request by SammyIsConfused**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [x] screenshots are included showing significant UI changes

##### Description of change
#1983 
- Removed indentation of p tags within manual data fix page
- Added consistent top and bottom padding to the buttons on the page
image

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Review by edalex-ian — COMMENTED** [review]
Looks good. However it looks like it still needs more space between the radio buttons and the button. Maybe those radio buttons need a margin or padding?

**Comment by SammyIsConfused** [comment]
> Looks good. However it looks like it still needs more space between the radio buttons and the button. Maybe those radio buttons need a margin or padding?

@edalex-ian Added in the padding, sorry about that. 
image

**Comment by SammyIsConfused** [comment]
image
Increased the font size of the radio button labels to match the other labels, and made the radio buttons side-by-side.

File: Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss [hunk 1]
```diff
@@ -5197,3 +5199,22 @@ label[for="password"] {
...
   outline: lightgrey dashed 1px;
 }
+
+.manualdatafixes {
+  button {
+    @include buttonShadowBorder;
+    margin-top: $singleMargin;
+    margin-bottom: $singleMargin;
+  }
+  #rmdfsgto_forceUpdate {
```
Comment (edalex-ian) [inline Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss:5209]: Wowsers! That's some ID hey? Obviously generated, so probably best to add a comment here to say what this is for. Is it something like:

```suggestion
  // The radio buttons for the 'Generate thumbnails and previews' section
  #rmdfsgto_forceUpdate {
```

**Review by edalex-ian — APPROVED** [review]
Great improvement @SammyIsConfused . If you could just add one comment for future maintenance that'd be great, and then good to merge on green.

**Pull request merged**